service/worker/archiver/archiver.go

```diff
@@ -131,6 +133,7 @@ func handleRequest(ctx workflow.Context, logger log.Logger, metricsClient metric
 			NonRetriableErrorReasons: deleteHistoryActivityNonRetryableErrors,
 		},
 	}
+	metricsClient.IncCounter(metrics.ArchiverScope, metrics.ArchiverDeleteWithRetriesCount)
```

Comment:
so effectively `ArchiverDeleteWithRetriesCount` will be equal to `ArchiverUploadWithRetriesCount` and be same as cadence requests..  I am ok with multiple metrics, but why do we need them?